routing/router.go

```diff
@@ -23,6 +24,29 @@ func NewRouter(conf *config.Config) *mux.Router {
 			Handler(route.HandlerFunc)
 	}
 
+	// All Jaeger Query routes are prefixed with /jaeger
+	// All querys type /jaeger/(.)* are redirect to <Jaeger Host>/api/ with Queryparams.
+	router.PathPrefix("/jaeger/{rest:.*}").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
What if Jaeger isn't deployed? Or is it supposed to always be?

Also for readability of the routes, maybe it's better to move out the handler function somewhere else (in handlers package?) and keep the route definition minimal here.

Reply:
Timeout I think

Reply:
Yeah but I mean, is it like for Grafana where we can't expect it to be present, and thus have a specific behaviour when it's not there?
Or do we assume Kiali needs Jaeger and thus cannot be installed without Jaeger?

(@theute I imagine this question is more for you)